test/functional-vanilla.bats

```diff
@@ -151,6 +151,43 @@ spec:
   done
 }
 
+@test "Rename Agents" {
+  initAgents
+  echo "========> Config.yaml before rename"
+  cat ~/.iofog/config.yaml
+  for IDX in "${!AGENTS[@]}"; do
+    local AGENT_NAME="${NAME}-${IDX}"
+    test iofogctl -v -n "$NS2" rename agent "$AGENT_NAME" "${AGENT_NAME}-test_rename"
+    echo "========> Config.yaml"
+    cat ~/.iofog/config.yaml
+    test iofogctl -v -n "$NS2" logs agent "${AGENT_NAME}-test_rename"
+    checkLegacyAgent "${AGENT_NAME}-test_rename" "$NS2"
```

Comment:
This only test that you have an agent with the new name, it does not check that the previous name is not showed anymore (I.E: offline ?)